client/app/common/authentication/authentication.js

```diff
@@ -1,10 +1,13 @@
 import Authentication from "./authentication.service";
 import uiRouter from "angular-ui-router";
+import { hookProvider } from "./hook.js";
```

Comment:
in this case I think default, and named export is perfect. for actually using it you would only use the default export, and for testing it handily exports the intermediate function.